pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -209,15 +209,15 @@ overscan_bottom=4
 
 ## Beaglebone
 
-Currently the Beaglebone devices are running a very new 4.1 kernel (which is obviously awesome), unfortunately many of the userspace libraries haven't caught up yet so they only work with the older 3.8 kernel. Luckily [ruth0000](https://github.com/ruth0000) was kind enough to patch the Octalbonescript JS library and made a lovely node.js module over here: https://www.npmjs.com/package/octalbonescript_capemgr4_1 .
+Currently the Beaglebone devices are running a v5.4 kernel (as of March 2021), unfortunately many of the userspace libraries haven't caught up yet so they only work with the older 3.8 kernel. Luckily [ruth0000](https://github.com/ruth0000) was kind enough to patch the Octalbonescript JS library and made a lovely node.js module over here: https://www.npmjs.com/package/octalbonescript_capemgr4_1 .
```

Comment:
@kenna-smith I would recommend not adding dates in documentation as it leads to redundancy much faster. Instead how about. 

`Currently, the Beaglebone devices are running a v5.4 kernel ([Check latest kernel version here](LINK TO KERNEL))`